lib/client-assets.php

```diff
@@ -830,11 +823,21 @@ function gutenberg_register_vendor_scripts() {
 	wp_add_inline_script( 'lodash', 'window.lodash = _.noConflict();' );
 	gutenberg_register_vendor_script(
 		'wp-polyfill-fetch',
-		'https://unpkg.com/whatwg-fetch/fetch.js'
+		'https://unpkg.com/whatwg-fetch/fetch.js',
+		array( 'wp-deprecated' )
+	);
+	wp_add_inline_script(
+		'wp-polyfill-fetch',
+		'wp.deprecated( "wp-polyfill-fetch script handle", { plugin: "Gutenberg", version: "4.0" } );'
```

Comment:
Does it mean that `wp-polyfill-ecmascript` contains also polyfill for fetch?

I can see some code for `Promise` in https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.0.0/polyfill.js, but can't see anything related to `fetch`.

Reply:
Since it relies on `core-js` I don't think it's included https://github.com/zloirock/core-js#missing-polyfills